test/System.Net.Http.Formatting.Test/Formatting/BsonMediaTypeFormatterTests.cs

```diff
@@ -330,7 +330,12 @@ public async Task FormatterThrowsOnReadWithInvalidContent()
             // Act & Assert
             await Assert.ThrowsAsync<JsonReaderException>(
                 () => formatter.ReadFromStreamAsync(variationType, stream, content, null),
-                "Could not convert to decimal: 7.92281625142643E+28. Path 'Value'.");
+#if NETCOREAPP3_1_OR_GREATER
```

Comment:
These are a little odd since you don't have a 3.1 TFM. I guess it's the most accurate since that's when the change happened?

Reply:
> These are a little odd since you don't have a 3.1 TFM. I guess it's the most accurate since that's when the change happened?

I also expanded toward `net6.0` incrementally. But the main thing was accuracy since someone might tweak a test project to include `netcoreapp3.1`.